WordPress/wporg-parent-2021
# Update posts and archives templates

See https://github.com/WordPress/wporg-main-2022/issues/266 — Rosetta sites use posts, so before rolling out the main theme, it needs to support posts & archives. This PR cleans up the templates, and moves some of the content out of the HTML templates into translatable php patterns.

### Screenshots

| | Before (large) | Before (small) | After (large) | After (small) |
|---|---|---|---|---|
| /news/ | ![before-news](https://github.com/WordPress/wporg-parent-2021/assets/541093/468ba077-7163-4020-9e9f-e30ed0069350) | ![before-news-small](https://github.com/WordPress/wporg-parent-2021/assets/541093/712abd70-0f08-4b52-9c66-fd00d9ad1634) | ![after-news](https://github.com/WordPress/wporg-parent-2021/assets/541093/a7392548-0cfb-4f3d-a5bf-b2e3e6e72227) | ![after-news-small](https://github.com/WordPress/wporg-parent-2021/assets/541093/c007d597-e3f4-4c47-8230-91b1dc1fdd1d) |
| Category | ![before-category](https://github.com/WordPress/wporg-parent-2021/assets/541093/259990e8-989b-4621-9fe3-8110c88739db) | ![before-category-small](https://github.com/WordPress/wporg-parent-2021/assets/541093/05e9d451-1b52-48b0-9651-68294f29663c) | ![after-category](https://github.com/WordPress/wporg-parent-2021/assets/541093/bd374abd-735d-49fb-9190-1a434cd685c1) | ![after-category-small](https://github.com/WordPress/wporg-parent-2021/assets/541093/d15d1a84-85be-41dc-b910-600bfe0d06bf) |
| Single | ![before-single](https://github.com/WordPress/wporg-parent-2021/assets/541093/d5f83105-3621-461d-91f2-9ecaff321414) | ![before-single-small](https://github.com/WordPress/wporg-parent-2021/assets/541093/403ec1da-c2a3-4ac4-826b-89e4d6fcb156) | ![after-single](https://github.com/WordPress/wporg-parent-2021/assets/541093/c95c2397-21ed-448f-a926-9cb446f6ae5e) | ![after-single-small](https://github.com/WordPress/wporg-parent-2021/assets/541093/13437ada-cba1-4ba9-85db-5bde8a5fe05a) |

### How to test the changes in this Pull Request:

1. Sync this branch to a sandbox, optionally plus this wporg-main-2022 PR (will add the local header).
2. Update `mu-plugins/rosetta-network/2022-theme-switcher.php` to enable the new theme — use [this file](https://gist.github.com/ryelle/f96403af237fddddd9e00b3bf91852d3), which also updates the Latest News block on the homepage.
3. View any rosetta site's `/news/` and click around though posts. For example
    - https://es.wordpress.org/news/ 
    - https://ar.wordpress.org/news/
    - https://ja.wordpress.org/news/
4. It should look good 🙂
